internal/cloudapi/v2/v2.go

```diff
@@ -127,6 +128,34 @@ func (h *apiHandlers) GetError(ctx echo.Context, id string) error {
 	return ctx.JSON(http.StatusOK, apiError)
 }
 
+// splitExtension returns the extension of the given file. If there's
+// a multipart extension (e.g. file.tar.gz), it returns all parts (e.g.
+// .tar.gz). If there's no extension in the input, it returns an empty
+// string. If the filename starts with dot, the part before the second dot
+// is not considered as an extension.
+func splitExtension(filename string) string {
```

Comment:
Is there a subtle difference to `path.Ext()` that makes this necessary? if so, that could be part of the documentation.

Reply:
this was carried over from the koji api unchanged

Reply:
path.Ext only retusn stuff after the last '.'; so it would return .gz